pkg/scheduler/framework/plugins/nodevolumelimits/csi.go

```diff
@@ -76,6 +76,8 @@ func (pl *CSILimits) Name() string {
 // failed by this plugin schedulable.
 func (pl *CSILimits) EventsToRegister() []framework.ClusterEventWithHint {
 	return []framework.ClusterEventWithHint{
+		// Adding CSINode gives a chance to schedule pods with CSI volumes.
+		// `QueueingHintFn` is not necessary because we aren't filtering by CSINodes.
```

Comment:
```suggestion
		// We don't register any `QueueingHintFn` intentionally
		// because any new CSINode could make pods that were rejected by CSI volumes schedulable. 
```